ros2component/ros2component/api/__init__.py

```diff
@@ -82,7 +81,10 @@ def get_container_components_info(*, node, remote_container_node_name):
         '{}/_container/list_nodes'.format(remote_container_node_name)
     )
     try:
-        list_nodes_client.wait_for_service()
+        if not list_nodes_client.wait_for_service(timeout_sec=5.0):
+            raise RuntimeError(
+                f'No list_nodes service found for {remote_container_node_name} container'
+            )
```

Comment:
I think this fix should be part of a different PR, or at least, don't squash the commit.

Reply:
Not planning to squash, that's why I pushed separate commits.